src/main/scala/com/workday/warp/math/linalg/AnomalyDetector.scala

```diff
@@ -8,23 +8,28 @@ import org.pmw.tinylog.Logger
 import scala.collection.mutable
 
 /**
-  * Thin wrapper around [[RobustPca]]. Holds penalty-related parameters, and implements double rpca.
+  * Thin wrapper around [[RobustPca]].
+  *
+  * Holds penalty-related parameters, implements double rpca.
+  *
+  * Performs Augmented Dickey-Fuller test to check for time series stationarity.
   *
   * Created by tomas.mccandless on 9/14/16.
   */
-case class RobustPcaRunner(lPenalty: Double = WARP_ANOMALY_RPCA_L_PENALTY.value.toDouble,
+case class AnomalyDetector(lPenalty: Double = WARP_ANOMALY_RPCA_L_PENALTY.value.toDouble,
```

Comment:
let me know what y'all think of this renaming. i thought the old name wasn't as suitable anymore since we have a new preprocessing step.

Reply:
hmm we could move the adf smoothing to a trait and mix that in here. This allows us to keep the name so if we intend to add more anomaly detectors in the future we won't have to refactor again

Reply:
cool. this smoothing thing is my own hacky invention separate from adf, but ill move it to a trait and revert this rename.